pse/p0tionConfig.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Ethstorage Circuits Ceremony",
+  "description": "This is a trusted setup ceremony for the Ethstorage Circuits with is used to proof storage on L2 dynamic datasets with an Ethereum L1 Contract",
+  "startDate": "2025-07-18T00:00:00",
+  "endDate": "2025-08-01T00:00:00",
+  "timeoutMechanismType": "DYNAMIC",
+  "penalty": "20",
+  "circuits": [
+      {
+          "description": "A simple blob encoder using Poseidon hash",
+          "compiler": {
+              "version": "2.0.6",
```

Comment:
How do we decide which Solidity compiler version to use? The latest release is 2.2.2, and the readme shows that `circom compiler 2.2.2 `

Reply:
update to v2.2.2